### PR TITLE
relnote(Fx115): String isWellFormed() and toWellFormed() behind pref in nightly

### PR DESCRIPTION
**Pull request by bsmth**
Nightly provides the pref `javascript.options.experimental.well_formed_unicode_strings` set to false (disabled) by default.

This allows for using the following methods:

- [`String.prototype.isWellFormed()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/isWellFormed)
- [`String.prototype.toWellFormed()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toWellFormed)

__Related issues and pull requests:__
- [x] Parent issue https://github.com/mdn/content/issues/27173
- [x] BCD - https://github.com/mdn/browser-compat-data/pull/20243

__Bugzilla:__
- BUG-1825005

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/string/iswellformed/index.md [hunk 1]
```diff
@@ -84,6 +84,6 @@ if (illFormed.isWellFormed()) {
 
 ## See also
 
-- [Polyfill of `String.prototype.isWellFormed` in `core-js`](https://github.com/zloirock/core-js#well-formed-unicode-strings)
 - {{jsxref("String.prototype.toWellFormed()")}}
 - {{jsxref("String.prototype.normalize()")}}
+- [Polyfill of `String.prototype.isWellFormed` in `core-js`](https://github.com/zloirock/core-js#well-formed-unicode-strings)
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/javascript/reference/global_objects/string/iswellformed/index.md:89]: I see, thanks for pointing it out! I will revert this

[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/javascript/reference/global_objects/string/iswellformed/index.md]: Done in 5eb1daff85f9e3a4aee353a810a0d9a81d653bd6

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -928,6 +928,48 @@ The `groupBy` method should be used when strings can be used to represent elemen
   </tbody>
 </table>
 
+### String isWellFormed() and toWellFormed() methods
+
+Strings in JavaScript are represented by unsigned 16-bit integers, and it's possible to have valid string types that are not well-formed UTF-16 Unicode text.
```
Comment (Josh-Cena) [inline files/en-us/mozilla/firefox/experimental_features/index.md:933]:
```suggestion
Strings in JavaScript are represented by sequences of unsigned 16-bit integers, and it's possible to have valid string values that are not well-formed Unicode text.
```

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 3]
```diff
@@ -928,6 +928,48 @@ The `groupBy` method should be used when strings can be used to represent elemen
...
+The {{jsxref("String.prototype.isWellFormed()")}} and {{jsxref("String.prototype.toWellFormed()")}} methods are used to check if a string is well-formed UTF-16 Unicode text and to convert a string to well-formed UTF-16 Unicode text, respectively.
```
Comment (Josh-Cena) [inline files/en-us/mozilla/firefox/experimental_features/index.md:934]:
```suggestion
The {{jsxref("String.prototype.isWellFormed()")}} and {{jsxref("String.prototype.toWellFormed()")}} methods are used to check if a string is well-formed Unicode text and to convert a string to well-formed Unicode text, respectively.
```

**Review by Josh-Cena — APPROVED** [review]
I'm not sure if you are waiting on my review, so leaving my approval, but you may want to get a Mozilla folk to merge this.

Also some wording suggestions.

**Comment by bsmth** [comment]
> I'm not sure if you are waiting on my review, so leaving my approval, but you may want to get a Mozilla folk to merge this.
> 
> Also some wording suggestions.

Thank you, Josh, I'll get another pair of eyes on it to merge 👍🏻

**Review by Rumyra — APPROVED** [review]

**Pull request merged**